### ISSUE TITLE
Incompatible with comments module

### ISSUE THREAD

**Issue by dhensby** [Firesphere/silverstripe-newsmodule#169]
This module is incompatible with the https://github.com/silverstripe/silverstripe-comments module due to a class naming collision.

Perhaps this module should use the comments module as a dependency?

**Comment by Firesphere** [comment]
Yep, I noticed this as well. Just haven't gotten around to fixing it. To not make it break on backward, I was thinking on refactoring towards NewsComments or something. Adapting the Comments module is a long-term fix.

**Comment by tungdt-90** [comment]
Hi !

I was the one who raised the issue on [silverstripe-comment](https://github.com/silverstripe/silverstripe-comments/issues/186#issuecomment-236524221).

In the meantime, i'm writing my own Comment object, but i would like to follow this issue, and hope that it can be fixed/merged soon.

Have good day !

**Comment by Firesphere** [comment]
Expect an update somewhere this week. I'm going to "namespace" all classes by prefixing them and add a backward compatibility build change. Stay on the stable branch please :)

**Comment by Firesphere** [comment]
Sorry it's taking so long. I'm running into some mild issues with the migration
